src/locale/sr-cyrl.js

```diff
@@ -110,6 +110,6 @@ export default moment.defineLocale('sr-cyrl', {
     ordinal: '%d.',
     week: {
         dow: 1, // Monday is the first day of the week.
-        doy: 7, // The week that contains Jan 7th is the first week of the year.
+        doy: 7, // The week that contains Jan 1st is the first week of the year.
```

Comment:
According to Serbian law, New Year is on the 1st of January since Serbia uses Gregorian calendar. Info about it can be found [here](https://www.paragraf.rs/propisi/zakon_o_drzavnim_i_drugim_praznicima_u_republici_srbiji.html) in article 1a.

Reply:
Same issue with `sr.js`.

Reply:
When this get's fixed, it'll affect the test cases so they should be changed.

Reply:
I am simply changing the comment. dow=1, doy=7 means the first week of the year includes 1st January (i.e.  7+dow-doy, see more information here https://momentjs.com/docs/#/customization/dow-doy/ )

Reply:
Sorry about that, my bad 😅. Everything else look great.